pkg/sync/v5/data/DaemonSet.apps/openshift-azure-monitoring/etcd-metrics.yaml

```diff
@@ -17,26 +17,25 @@ spec:
     spec:
       containers:
       - args:
-        - -loglevel
+        - tlsproxy
```

Comment:
```
command:
- tlsproxy
```
? 

comment applies throughout PR

Reply:
https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#notes command is forced from image "entrypoint" `azure tlsproxy`:
this would be : 
```
command: azure
args: 
 - tlsproxy
 - params
```
But now azure is handled on image level